messages/en.json

```diff
@@ -289,7 +289,7 @@
   "admin/b2b-organizations.organization-settings-admin.customFieldsTitle": "Custom Fields",
   "admin/b2b-organizations.custom-fields.title": "Custom Fields",
   "admin/b2b-organizations.custom-fields.titleSingular": "Custom Field",
-  "admin/b2b-organizations.custom-fields.explanation": "Custom fields are used to add additional information to organizations. Internal use only, will not be shown to the customer.",
+  "admin/b2b-organizations.custom-fields.explanation": "Custom fields are used to add additional information to organizations or cost centers. Internal use only, will not be shown to the customer.",
```

Comment:
The "Internal use only" part is no longer true in all cases

Reply:
Good catch, fixed messaging in all languages